dusty/cli/test.py

```diff
@@ -2,13 +2,14 @@
 If args are passed, default arguments are dropped
 
 Usage:
-  test [options] <app_or_lib_name> [<suite_name>] [<args>...] [--recreate] [--no-pull]
+  test [options] <app_or_lib_name> [--all] [<suite_name>] [<args>...]
```

Comment:
Does docopt give us a way to say you can either provide --all or you can provide a suite_name with args?